### PR TITLE
Creates new directory focus with 3 blank files and new folder

### PR DESCRIPTION
**Pull request by GHJon**

### CODE REVIEW

**Review by jmuratov98 — APPROVED** [review]

File: tree*.c [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+        if (mkdir(dirname[1], S_IRWXU) == -1){
+            perror("Error: \n");
+        }
+        else printf("Directory %s Created\n", dirname[1]);
+   }
+    else printf("Failed Opening %s", dirname[0]);
+
+    return 0;
+}
```
Comment (JiejayLan) [inline tree*.c:58]: need a new line at the end

File: tree*.c [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+			    return 1;
+		    }
+		    else printf("Creation of %s is successful \n", filenames[2]);
+
+        if (mkdir(dirname[1], S_IRWXU) == -1){
+            perror("Error: \n");
+        }
+        else printf("Directory %s Created\n", dirname[1]);
+   }
+    else printf("Failed Opening %s", dirname[0]);
```
Comment (JiejayLan) [inline tree*.c:55]: The code for create 3 files are duplicate,  can use a for loop to refactor the logic

**Review by JiejayLan — CHANGES_REQUESTED** [review]
Check my comments, the logic should not have big issues

**Review by JiejayLan — APPROVED** [review]
Nothing I can say

**Pull request merged**